tomee/tomee-mojarra/src/main/resources/META-INF/openwebbeans/openwebbeans.properties

```diff
@@ -0,0 +1,22 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+configuration.ordinal=75
+
+# We need to workaround an integration issue between Mojarra and OWB right now
+# For more details see https://github.com/apache/openwebbeans/pull/89
+# and https://github.com/eclipse-ee4j/mojarra/issues/5457
+org.apache.webbeans.ignoredExtensions=com.sun.faces.cdi.CdiExtension
```

Comment:
I'd rather not use it since the values are not merged so it means users can't rely on that anymore